src/main/scala/io/shiftleft/js2cpg/core/Report.scala

```diff
@@ -82,7 +82,7 @@ class Report {
         s"${reports.map(_._2.loc).sum}",
         s"${reports.count(_._2.parsed)}/$numOfReports",
         s"${reports.count(_._2.cpgGen)}/$numOfReports",
-        s"${TimeUtils.pretty(Duration.fromNanos(reports.map(_._2.duration).sum))}"
+        ""
```

Comment:
Reason to remove this here: calculating the runtime sum does not make sense when running CPG creation for files in parallel.